src/main/get-metrics.injectable.ts

```diff
@@ -22,13 +25,16 @@ const getMetricsInjectable = getInjectable({
     ) => {
       const prometheusPrefix = cluster.preferences.prometheus?.prefix || "";
       const metricsPath = `/api/v1/namespaces/${prometheusPath}/proxy${prometheusPrefix}/api/v1/query_range`;
+      const body = new FormData();
+
+      for (const [key, value] of object.entries(queryParams)) {
```

Comment:
Why `object.entries` and not standard `Object.entries`? What the difference here?

Reply:
Because the `Object.entries` is not typed as usefully. The only difference is their typing.